examples/react/src/App.js

```diff
@@ -391,6 +405,26 @@ renderLoginButtons() {
           logoStyle={{marginLeft:0}}
           onClick={()=>this.handleLogin("lynx")}
       />
+      <LoginButton provider='portis'
+          buttonStyle={{width:250, marginTop:'24px'}}
+          logoStyle={{marginLeft:0}}
+          onClick={()=>this.handleLogin("portis")}
+      />
+      <LoginButton provider='whalevault'
+          buttonStyle={{width:250, marginTop:'24px'}}
+          logoStyle={{marginLeft:0}}
+          onClick={()=>this.handleLogin("whalevault")}
+      />
+      <LoginButton provider='simpleos'
```

Comment:
I get "Failed to connect to simpleos wallet: TypeError: Failed to fetch" 
Don't think we should ever get this error - unless it's in the provider doing the fetch. Is that where it is trying to fetch?

Reply:
Yes that error is inside of the provider. I've tried with http/https but it doesn't make a difference. It's worth noting that the simpleos website says this functionality is in beta.